### PR TITLE
Output Issuer ID instead of [object Object].

### LINKED ISSUE #14: Wallet store() event screen doesn't display Issuer details
It only shows, `[object Object]`

image

### PR DESCRIPTION
**Pull request by BigBlueHat**
Fixes #14.

Not a thorough fix--as the ID is pretty paltry in terms of data, but it at least looks better than `[object Object]`.

### CODE REVIEW

File: wallet-ui-store.html [hunk 1]
```diff
@@ -94,7 +94,7 @@ <h6>Wallet Contents:</h6>
       ? vp.verifiableCredential[0]
       : vp.verifiableCredential;
     document.getElementById('credentialType').innerHTML = getCredentialType(vc);
-    document.getElementById('credentialIssuer').innerHTML = vc.issuer;
+    document.getElementById('credentialIssuer').innerHTML = vc.issuer.id;
```
Comment (dlongley) [inline wallet-ui-store.html:97]: Hmm, needs to be more like:

```suggestion
    document.getElementById('credentialIssuer').innerHTML = vc.issuer.id ?? vc.issuer;
```

`issuer` can be a string with the ID value or an object that MUST have an `id` property with the ID as the value.

**Review by dlongley — CHANGES_REQUESTED** [review]

**Comment by BigBlueHat** [comment]
@dlongley good catch! Sorry I missed that. Fixed now.

**Review by dlongley — APPROVED** [review]

**Comment by dlongley** [comment]
Also, this will need a CHANGELOG entry.

**Comment by BigBlueHat** [comment]
@dlongley added a changelog entry. If you're cool with it, let me know or just merge. 😃

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # chapi-demo-wallet Changelog
 
+## 2.0.4 - 2023-08-16
```
Comment (dlongley) [inline CHANGELOG.md:3]: JFYI, this has to be set right before running the tools (the tool needs to see an uncommitted change in the changelog, where we put the release date), so we typically just leave a placeholder for the day until release time.

```suggestion
## 2.0.4 - 2023-08-dd
```

**Review by dlongley — APPROVED** [review]

**Pull request merged**